src/main/java/xyz/nucleoid/creator_tools/command/MapManageCommand.java

```diff
@@ -339,14 +339,16 @@ private static int deleteWorkspace(CommandContext<ServerCommandSource> context)
 
         var workspaceManager = MapWorkspaceManager.get(source.getServer());
 
-        MutableText message;
-        if (workspaceManager.delete(workspace)) {
-            message = Text.translatable("text.nucleoid_creator_tools.map.delete.success", workspace.getIdentifier());
-        } else {
-            message = Text.translatable("text.nucleoid_creator_tools.map.delete.error", workspace.getIdentifier());
-        }
+        source.sendFeedback(() -> {
+            MutableText message;
+            if (workspaceManager.delete(workspace)) {
```

Comment:
This code won't run when output is suppressed - actual logic shouldn't be happening within here 😄

Reply:
Fixed in 571abef86b28441f2a9c8eefe22a26ba931a97f8